VideoWeb/VideoWeb/ClientApp/src/app/services/events.service.ts

```diff
@@ -238,6 +238,14 @@ export class EventsService {
         this._handlersRegistered = false;
     }
 
+    get eventHubIsConnected(): boolean {
+        return this.eventsHubService.isConnectedToHub;
+    }
+
+    onEventsHubReady(): Observable<any> {
+        return this.eventsHubService.onEventsHubReady;
+    }
```

Comment:
Is it necessary to have both?

Reply:
Yes, I checked again and I think having both is needed.